source-code/core/src/lint/linter.ts

```diff
@@ -64,7 +57,7 @@ const processLintRule = async ({
 
 	const context = createContext(id, level)
 
-	const payload = await setup({ env, referenceLanguage, languages, context })
+	const payload = await setup({ referenceLanguage, languages, context })
```

Comment:
I think it was one of the first suggestion from you that the lint functionality might import something else.
But we can see if it is really needed and add the funcitonality later if someone asks for it.

Reply:
I assumed that the env exposing might come from importing code. I referred to `fetch` though to make API calls. fetch is available regardless of env